docker-compose.yml

```diff
@@ -62,20 +53,21 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
 
   localstack:
-    image: localstack/localstack:0.11.2
+    image: localstack/localstack:0.12.10
     networks:
       - hmpps
     container_name: localstack
     ports:
       - "4566-4597:4566-4597"
       - 8999:8080
     environment:
-      - SERVICES=sqs,sns,es
+      - SERVICES=sqs,sns,es,elasticsearch
```

Comment:
I thought es was elasticsearch... what's es?

Reply:
I think I added this when hacking around to get working, think can remove now